go/libkb/keyfamily.go

```diff
@@ -728,6 +739,36 @@ func (ckf ComputedKeyFamily) GetActivePGPKeys(sibkey bool) (ret []*PGPKeyBundle)
 	return
 }
 
+type RevokedKey struct {
+	Key       GenericKey
+	RevokedAt *KeybaseTime
+}
+
+func (ckf ComputedKeyFamily) GetRevokedKeys() []RevokedKey {
+	var revokedKeys []RevokedKey
+	for kid := range ckf.kf.AllKIDs {
+		ki := ckf.cki.Infos[kid]
+		if ki == nil {
+			continue
```

Comment:
I feel like this branch should be an error or a panic too. Something would have to be terribly wrong here, right?